lib/crowbar/client/command/backup/upload.rb

```diff
@@ -19,6 +19,17 @@ module Command
       module Backup
         class Upload < Base
           def request
+            case args.file
```

Comment:
Style/ConditionalAssignment: Use the return of the conditional for variable assignment and comparison.